css/global.css

```diff
@@ -2,3 +2,8 @@
 .p-channel_sidebar__banner--unreads, .p-workspace_banner__desktop-notifications, .c-mention_badge {
   display: none !important;
 }
+
+
+.c-message_kit__hidden_message_blur {
+  filter: blur(0) !important;
```

Comment:
In dev tools typically I'll just disable css like this, but according to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/blur) you can remove a blur effect by passing the `blur()` function a 0 for the radius instead of a larger number.